### PR TITLE
Describe the Recently Ordered block for the My Account page

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request describes the Recently Ordered block for the My Account page

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/customers/account-dashboard-my-account.html

### CODE REVIEW

File: src/customers/account-dashboard-my-account.md [hunk 1]
```diff
@@ -6,3 +6,12 @@ The My Account section of the customer dashboard gives customers an overview of
 
 ![]({% link images/images-ee/customer-account-dashboard-my-account.png %}){: .zoom}
 _My Account_{:.ee-only}
+
+## Recently Ordered Products
+
+The **Recent Orders** block is displayed in the sidebar and on the **My Account page**, this feature is available only for customers who are logged in. The **Recently ordered** block appears after placing an order, it displays 5 products from the last purchase.
+
+![]({% link images/images/recently-ordered.png %}){: .zoom}
```
Comment (dobooth) [inline src/customers/account-dashboard-my-account.md:14]: Please remove this image.

**Review by dobooth — APPROVED** [review]

File: src/customers/account-dashboard-my-account.md (hunk 1 above)
Comment (ZhovnirMaria) [reply src/customers/account-dashboard-my-account.md]: @dobooth 
Done, thanks 😊

File: src/customers/account-dashboard-my-account.md [hunk 2]
```diff
@@ -6,3 +6,9 @@ The My Account section of the customer dashboard gives customers an overview of
 
 ![]({% link images/images-ee/customer-account-dashboard-my-account.png %}){: .zoom}
 _My Account_{:.ee-only}
+
+## Recently Ordered Products
+
+The **Recent Orders** block is displayed in the sidebar and on the **My Account page**, this feature is available only for customers who are logged in. The **Recently ordered** block appears after placing an order, it displays 5 products from the last purchase.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-account.md:12]:
```suggestion
The **Recent Orders** block is displayed in the sidebar and on the **My Account page** for customers who are logged in. The **Recently ordered** block appears after placing an order, it displays five products from the last purchase.
```

File: src/customers/account-dashboard-my-account.md [hunk 3]
```diff
@@ -6,3 +6,9 @@ The My Account section of the customer dashboard gives customers an overview of
...
+
+The customer can re-add products to the cart by checking the products and clicking on the **Add to Cart** button. You can also view the last order by clicking the **View all** button, the customer will be redirected to the **My account** page, to the **Recent Orders** block.
```
Comment (leslietilling) [inline src/customers/account-dashboard-my-account.md:14]: Let's keep whole thing in third person. Also some edits for grammar and style

```suggestion
The customer can re-add products to the cart by selecting the products and clicking **Add to Cart**. They can also view the last order by clicking **View all**, which redirects to the _My account_ page to includes the **Recent Orders** block.
```

Comment (ZhovnirMaria) [reply src/customers/account-dashboard-my-account.md:14]: @leslietilling thanks, done.

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1184`.

**Pull request merged**